explorer-api/src/country_statistics/mod.rs

```diff
@@ -51,17 +52,35 @@ impl CountryStatistics {
 
         info!("Locating mixnodes...");
         for (i, bond) in mixnode_bonds.iter().enumerate() {
```

Comment:
Since you only seem to care about values of the map, you might want to iterate via `mixnode_bonds.values()` to get rid of those `.1` uses